app/views/lists/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= content_tag_for(:section, @list) do %>
+  <h2><%= @list.title %></h2>
+  <p><%= @list.description %></p>
+  <ul>
+    <%= content_tag_for(:li, @list.books) do |book| %>
+        <!-- TODO: add link_to book show -->
+        <%= image_tag book.cover_image_url, alt: book.title, id: "book-#{book.id}" %>
```

Comment:
Good point, I rebased after putting this comment. I will get on this one with the next update.